deploy/kube-config/riemann/heapster-service.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Can we place a symlink to an existing service file instead of creating a new one?